job-server-tests/src/main/java/spark.jobserver/JavaHelloWorldJob.java

```diff
@@ -1,12 +1,12 @@
-package spark.jobserver;
+// package spark.jobserver;
```

Comment:
Yes, but I think it should be a separate pull request since the java api is gonna need some project restructuring since java is picky about what is located where.